ui/store/actions.ts

```
@@ -5085,6 +5086,39 @@ export function setName(
   }) as any;
 }
 
+/**
+ * To create a data deletion regulation for MetaMetrics data deletion
+ */
+export function createMetaMetricsDataDeletionTask(): ThunkAction<
+  void,
+  MetaMaskReduxState,
+  unknown,
+  AnyAction
+> {
+  return async (dispatch: MetaMaskReduxDispatch) => {
+    try {
+      await submitRequestToBackground('createMetaMetricsDataDeletionTask');
+      await forceUpdateMetamaskState(dispatch);
+    } catch (error) {
+      logErrorWithMessage(error);
```

Why are we suppressing errors here?

---

Likewise for `checkDataDeletionTaskStatus`